### PR TITLE
Add support for strawberry-graphql>=0.69

### PR DESCRIPTION
**Pull request by KundaPanda**

### CODE REVIEW

**Comment by patrick91** [comment]
@KundaPanda thanks for doing this! I'll try to review this by end of the week :)

**Comment by edouardruiz** [comment]
Hi there, I was wondering if there's anything blocking this review. I'm trying to use both [strawberry-graphql-django](https://github.com/strawberry-graphql/strawberry-graphql-django) and [strawberry-django-jwt](https://github.com/KundaPanda/strawberry-django-jwt) and some features (like the schema export) seem broken with the old version of strawberry.

**Comment by patrick91** [comment]
Sorry I've been busy last, I'll try to review soon!

File: strawberry_django/filters.py [hunk 1]
```diff
@@ -157,6 +159,8 @@ def get_filters(self):
         if not is_unset(self.filters):
             return self.filters
         type_ = self.type or self.child.type
+        while isinstance(type_, StrawberryContainer):
+            type_ = type_.of_type
```
Comment (patrick91) [inline strawberry_django/filters.py:163]: we should probably make this into a function/method. /cc @BryceBeagle

File: tests/auth/test_queries.py [hunk 2]
```diff
@@ -9,7 +11,7 @@
 
 @strawberry.type
 class Query:
-    current_user: User = auth.current_user()
+    current_user: Optional[User] = auth.current_user()
```
Comment (patrick91) [inline tests/auth/test_queries.py:14]: 👍 that solves my comment above :)

**Review by patrick91 — COMMENTED** [review]
this looks good to me, awesome work! I'll try to test this over the weekend

File: strawberry_django/fields/field.py [hunk 3]
```diff
@@ -1,34 +1,38 @@
...
+            type_ = type_.of_type
         get_queryset = getattr(type_, 'get_queryset', None)
         if get_queryset:
             queryset = get_queryset(self, queryset, info, **kwargs)
         return queryset
 
     @property
     def django_model(self):
-        type_ = self.type or self.child.type
+        type_ = (self.type.of_type if isinstance(self.type, StrawberryContainer) else self.type) or self.child.type
```
[the remark this answers is not in the document]
Comment (KundaPanda) [reply strawberry_django/fields/field.py]: Yes, it should. I forgot to update this type unwrapping.

**Review by patrick91 — APPROVED** [review]

**Comment by patrick91** [comment]
@KundaPanda thanks for this PR. I've just merged and released: https://pypi.org/project/strawberry-graphql-django/0.2.5/

I'll invite you to the organisation! Let me know if you want a sticker by filling this form https://forms.gle/dmnfQUPoY5gZbVT67

**Pull request merged**